### PR TITLE
Feature/test coverage reporting

### PR DESCRIPTION
**Pull request by kvithayathil**
Generate Jacoco reports e.g. `./gradlew clean build -Pbuild=dev jacocoTestReport
`

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -31,6 +31,10 @@ install:
...
 
-script: ./gradlew clean build -Pbuild=dev
-script: ./gradlew test -Pbuild=dev
+script:
+  - ./gradlew clean build -Pbuild=dev jacocoTestReport
+  - ./gradlew test -Pbuild=dev
 # script: ./gradlew connectedAndroidTest -Pbuild=dev
+
+#after_success:
+#  - bash <(curl -s https://codecov.io/bash)
```
Comment (TheHollidayInn) [inline .travis.yml]: Why is this removed?

**Review by TheHollidayInn — COMMENTED** [review]
Looks great! I assume this automatically generates the coverage locally as well or no?

File: .travis.yml (hunk 1 above)
Comment (kvithayathil) [reply .travis.yml]: Just in case it breaks the build since I'm not sure if CodeCov has been enabled yet

Comment (TheHollidayInn) [reply .travis.yml]: You should be able to run this now.

**Comment by kvithayathil** [comment]
Re-enabled code-cov in configuration.

Also, it should generate Jacoco reports locally - both as html and xml
